src/lug/Window/Window.cpp

```diff
@@ -2,37 +2,49 @@
 
 #if defined(LUG_SYSTEM_WINDOWS)
 
-    #include <lug/Window/Win32/WindowImplWin32.hpp>
+#include <lug/Window/Win32/WindowImplWin32.hpp>
 
 #elif defined(LUG_SYSTEM_LINUX)
 
-    #include <lug/Window/Unix/WindowImplX11.hpp>
+#include <lug/Window/Unix/WindowImplX11.hpp>
 
 #elif defined(LUG_SYSTEM_ANDROID)
 
-    #include <lug/Window/Android/WindowImplAndroid.hpp>
+#include <lug/Window/Android/WindowImplAndroid.hpp>
+
+#else
+
+// Theorically this should never happen since the Config.cmake will
+// warn the user before, but #error anyway
+#error "Unsupported operating system or environment"
 
 #endif
 
-lug::Window::Window::Window() : _impl{new lug::Window::priv::WindowImpl(this)} {
+namespace lug {
+namespace Window {
+
+Window::Window() : _impl{ new priv::WindowImpl(this) } {
+    initKeyState();
 }
 
-lug::Window::Window::Window(Window&& rhs) : _impl{rhs._impl} {
+Window::Window(Window&& rhs) : _impl{rhs._impl} {
```

Comment:
If think that we can't delete movable operators from `Window` since we handle them exclusively with `std::unique_ptr`